runtime/onert/core/include/compiler/train/LoweredTrainableGraph.h

```diff
@@ -68,7 +68,7 @@ class LoweredTrainableGraph : public ILoweredGraph
    *  @note   It uses copy of graph, not reference.
    *          It allows the original graph can be compiled multiple times.
    */
-  ir::train::TrainableGraph &_trainable_graph;
+  ir::train::TrainableGraph _trainable_graph;
```

Comment:
@jyoungyun or @ragmani 

I considered #11219 is for fixing a bug. But this change and the comment above says it is for allowing multiple time compilation. When do we need to compile multiple times for a given TrainableGraph?

Reply:
Both commits are required. The `_trainable_graph` disappeared after Executor was initialized. (I don't know the exact time.) Therefore, the reference value cannot be used after compilation step is finished. The deep copy are useful when this graph is copied to the TrainableExecutor.

Reply:
@jyoungyun Thank you for explanation. But my curiosity is mainly about _"can be compiled multiple times"_ from the comment.

```cc
 @note   It uses copy of graph, not reference.
         It allows the original graph can be compiled multiple times.
```